index.js

```diff
@@ -1 +1,34 @@
-// Code your solution in this file!
+const drivers = ["Antonia", "Nuru", "Amari", "Mo"];
+
+const returnFirstTwoDrivers = function () {
+  return drivers.slice(0, 2);
+};
+returnFirstTwoDrivers(drivers);
+
+const returnLastTwoDrivers = function () {
+  const lastTwo = drivers.slice(-2);
+  return lastTwo;
+};
+returnLastTwoDrivers(drivers);
+
+const selectingDrivers = [returnFirstTwoDrivers, returnLastTwoDrivers];
+
+function createFareMultiplier(num) {
+  return function (fare) {
+    return num * fare;
+  };
+}
+
+const fareDoubler = function (num2) {
+  createFareMultiplier();
+  return num2 * 2;
+};
```

Comment:
You have the fare multiplier, but in your implementation it's not actually being used. You can do it like this as well:
```suggestion
const fareDoubler = function (num2) {
  return createFareMultiplier(2);
};
```

Reply:
Thanks!